src/explore-education-statistics-common/src/services/permalinkSnapshotService.ts

```diff
@@ -0,0 +1,51 @@
+import { TableJson } from '@common/modules/table-tool/utils/mapTableToJson';
+import { dataApi } from '@common/services/api';
+import { TableDataQuery } from '@common/services/tableBuilderService';
+import { Footnote } from '@common/services/types/footnotes';
+import { UnmappedTableHeadersConfig } from '@common/services/permalinkService';
+
+export interface PermalinkSnapshot {
+  created: string;
+  dataSetTitle: string;
+  id: string;
+  publicationTitle: string;
+  status:
+    | 'Current'
+    | 'SubjectRemoved'
+    | 'SubjectReplacedOrRemoved'
+    | 'NotForLatestRelease'
+    | 'PublicationSuperseded';
+  table: {
+    caption: string;
+    footnotes: Footnote[];
+    json: TableJson;
+  };
+}
+
+interface CreatePermalink {
+  query: TableDataQuery;
+  configuration: {
+    tableHeaders: UnmappedTableHeadersConfig;
+  };
+}
+
+export default {
```

Comment:
Would suggest we make this a named export as It'll soon become a lint error (via EES-3409)